disperser/apiserver/server.go

```diff
@@ -311,6 +318,38 @@ func (s *DispersalServer) disperseBlob(ctx context.Context, blob *core.Blob, aut
 	}, nil
 }
 
+func (s *DispersalServer) DispersePaidBlob(ctx context.Context, req *pb.DispersePaidBlobRequest) (*pb.DisperseBlobReply, error) {
```

Comment:
Should we put this code behind a feature flag? Right now, if we were to release from this code, we'd be running code here which is still incomplete. (The previous PR hits the unimplemented fallback)

Reply:
:thinking: like if `paymentFeatureEnabled` do this, otherwise hit unimplemented fallback? the main incomplete part is the contract, could we wait til code complete to make a release?

Reply:
Could a feature flag be useful for rollout plan? I'd imagine we might want to test this on preprod and testnet before mainnet? cc @ian-shim

Reply:
sounds good to me:) I added a check with if `meterer` is nil then the function fallback. This is because meterer is only set when `EnablePaymentMeterer` is true, and wanted to avoid adding additional config to DisperserServer